.circleci/config.yml

```diff
@@ -997,16 +1012,20 @@ jobs:
       - run:
           name: "Yarn: Install Dependencies"
           command: yarn install --frozen-lockfile --non-interactive
+
       - save_cache:
           key: *windows_yarn_cache_key
           paths:
             - C:\Users\circleci\AppData\Local\Yarn
 
-      # Try to install the SDK up to 3 times, since network flakiness can cause install failures
-      # Using a timeout of 9 mins, as circle ci will timeout if there is no output for 10 mins
       - run:
           name: Install Android SDK Tools
-          command: choco install android-sdk --timeout 540; if (!$?) { choco install android-sdk --timeout 540 --force --forcedependencies}; if (!$?) { choco install android-sdk  --force --forcedependencies}
+          command: choco install android-sdk;
+
+      - save_cache:
+          key: *windows_choco_cache_key
+          paths:
+            - $env:CHOCO_CACHE_DIR
```

Comment:
Can we move this to just after the choco install?

Reply:
I'm not sure how you mean?  Isn't this being called directly after `choco install android-sdk`?

Reply:
You're actually right :) Can I ask you to move both this step AND the `name: Install Android SDK Tools` above. Ideally we can also have a single Choco install (it will make the CircleCI setup smaller and easier to maintain).